apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -563,45 +562,48 @@ export class TriggerEvent {
       const errorId = 'mail_unexpected_error';
 
       if (!email) {
-        const mailErrorMessage = `${errorMessage} email adress`;
+        const mailErrorMessage = `${errorMessage} email address`;
 
-        await this.sendMailStatus(
+        await this.sendErrorStatus(
           message,
           status,
           errorId,
           mailErrorMessage,
           command,
           notification,
           subscriber,
-          template
+          template,
+          LogCodeEnum.SUBSCRIBER_MISSING_EMAIL
         );
       }
       if (!integration) {
-        const integrationError = `${errorMessage} active integration  was found`;
+        const integrationError = `${errorMessage} active email integration`;
```

Comment:
Maybe worth adding some text here in the end of the message?
```suggestion
        const integrationError = `${errorMessage} active email integration not found`;
```

If logically it make sense